app/assets/javascripts/student_profile_v2/attendance_details.js

```diff
@@ -7,6 +7,31 @@
   var ProfileChart = window.shared.ProfileChart;
   var Scales = window.shared.Scales;
 
+  var styles = {
+    box: {
+      border: '1px solid #eee',
+      padding:'15px',
+      marginTop: '10px',
+      marginBottom: '10px',
+      width: '50%'
+    },
+    item: {
+      width: '33%',
+      display: 'inline-block',
+      paddingBottom: '2%'
+    },
+    itemHead: {
+      fontWeight: 'bold',
+    },
+    header: {
+      padding: '5%',
```

Comment:
For padding and margins, let's use actual pixel numbers since these don't need to be fully responsive.  In my experience making small styling changes like look good in a fully responsive is a lot of work with not much benefit to the end user. :)